lib/fluent/plugin/in_monitor_agent.rb

```diff
@@ -364,6 +368,13 @@ def get_monitor_info(pe, opts={})
           }
         end
         obj['instance_variables'] = iv
+      elsif ivars = opts[:ivars]
+        iv = {}
+        ivars.each {|name|
+          iname = :"@#{name}"
```

Comment:
@repeatedly I chose symbol literal instead of string because reduce garbage. Is this too careful?